Round 2 edits

gtw2 opened the pull request:
Using this as a place to compile edits.
- [x] add table of all S&O and parameters
- [x] grammar fixes (still a few)
- [x] remove algorithms
- [x] Need to move process detail
- [x] add approx. same amount of detail to all processes (volox. may need more?) 
- [x] citation fixes
- [x] explain table
- [x] smaller flowsheets for the 4 subprocesses

katyhuff committed:
adds makefile and acronyms

katyhuff committed:
better use of acronyms and use of builtin excellent features of latex (like figure counting)

gtw2 committed:
List of S&O

adds table and a few edits

gtw2 committed:
Revert "List of S&O"

This reverts commit b29cbdb10f8c66dd9e1e4599ca64fa749cf9fd1d.

gtw2 committed:
List of S&O

adds table of  parameters and a few edits. does not include build files this time

gtw2 committed:
first page and algorithm cleanup

edits to page one, removal of proposed algorithms

gtw2 committed:
rearrange

moved process discussion to respective sections.

Added more volox detail to chart.

fixed some citations, added volox source.

edits

gtw2 commented:
Should I cite papers inside the table of parameters?

gtw2 committed:
more moving

added to volox, moved reduction and refining sections

made flowsheet black and white

gtw2 committed:
material balance

adds volox and winning material balance (mini flowsheet)

gtw2 committed:
flowsheets

adds small flowsheets

adds summaries for some flowsheets and table

katyhuff commented:
> Should I cite papers inside the table of parameters?

Yes, if the parameter value came from somewhere you should cite that. A common way is to have a table structure where one of the columns is "Ref." and contains references where the parameters were found.

gtw2 committed:
last round 2 edits

fixed citations
added refs to table
added table discussion
fixed reduction image

gtw2 committed:
word choice

changed senteces

katyhuff commented:
@gtw2 This branch has conflicts. Please resolve them.

katyhuff commented:
Do not version this file: reduction.synctex.gz

katyhuff commented on winning.tex:
```diff
@@ -0,0 +1,42 @@
+% Author: Kathryn Huff
```
I am not the author of this file. Please amend. Say "Authors: Kathryn Huff, Gregory Westphal" or similar.

katyhuff commented on volox.tex:
```diff
@@ -0,0 +1,42 @@
+% Author: Kathryn Huff
```
I am not the author of this file. Please amend. Say "Authors: Kathryn Huff, Gregory Westphal" or similar.

katyhuff commented:
Generally, versioning pdfs is not advised.

katyhuff commented on 2018-westphal-antpc.tex:
```diff
@@ -1,11 +1,11 @@
-\title{PyRe: A Cyclus Pyroprocessing Facility Archetype}
-\author{\textbf{Gregory T. Westphal} and Kathryn D. Huff}
+\title{Pyre: A Cyclus Pyroprocessing Facility Archetype}
+\author{Gregory T. Westphal* and Kathryn D. Huff}
```
$^*$

katyhuff commented on 2018-westphal-antpc.tex:
```diff
@@ -1,11 +1,11 @@
 \institute{
 Dept. of Nuclear, Plasma and Radiological Engineering, University of Illinois at Urbana-Champaign \\
-gtw2@illinois.edu
+*gtw2@illinois.edu
```
$^*$

katyhuff committed:
2018-westphal-antpc.tex had some conflicts. Hopefully solved..

katyhuff commented on 2018-westphal-antpc.tex:
```diff
@@ -73,159 +77,175 @@ \section{Background: Pyroprocessing}
 \section{Method: \Cyclus Simulation}
-The separations facility provided by the \Cycamore library expansion is used as an initial model of a simple PRIDE facility. 
-The separations archetype allows for the declaration of a feed stream and requires the user-definition of facility efficiencies. 
+The separations facility provided by the \Cycamore library is used as an initial model of a simple PRIDE facility. 
```
The pride facility has not been explained and KAERI is no longer mentioned or cited (those parts were deleted?). This is not good.

katyhuff committed:
stop versioning pdfs

katyhuff commented:
please see #15 for a pr that fixes some of these issues.

katyhuff committed:
adds some acros

gtw2 committed:
Merge pull request #15 from katyhuff/issue11

Issue11

katyhuff commented:
If you can fix the inline comments in the above review, this can be pulled. Once this is merged, then please move on to #17 .

gtw2 committed:
fix author

katyhuff commented:
![bitmoji](https://render.bitstrips.com/v2/cpanel/8c15569e-920d-40ca-a005-52d026c9bde5-aef391d1-ee2a-4ffc-b1c7-1c24444a7cae-v1.png?transparent=1&palette=1&width=246)
